modules/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ include(FetchContent)
 FetchContent_Declare(
   SPIRV-Headers
   GIT_REPOSITORY https://github.com/KhronosGroup/SPIRV-Headers.git
-  GIT_TAG        sdk-1.3.239.0
+  GIT_TAG        e77d030
```

Comment:
Yeah, I dunno what's best. It's not even LLVM, it's DPC++ which includes its own copy of [SPIRV-LLVM-Translator)](https://github.com/KhronosGroup/SPIRV-LLVM-Translator) which includes SPIRV-Headers. DPC++ are ultimately in control of which `SPIRV-LLVM-Translator` commits they cherry-pick into DPC++.

Either way I can't see a way of automagically grabbing any data about what commit they've settled on.

Ideally it should never be too much a problem, but the backwards-compatible change they landed to the headers means we can only sit on one side of that fence. There's no data in `SPIRV-Headers` itself that I can see which would allow us to conditionally compile to adapt both.

Currently SPIRV-LLVM-Translator is sat on [1c6bb2743599e6eb6f37b2969acc0aef812e32e3](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/spirv-headers-tag.conf), as is [DPC++](https://github.com/intel/llvm/blob/sycl/llvm-spirv/spirv-headers-tag.conf). That's older than the commit I've chosen.

Maybe this'll get better if/when upstream LLVM supplies SPIRV-Headers itself? Not sure what the plan there is, with regards to DPC++ merging into LLVM.

Reply:
I've updated for us to use that commit too.